rs/sns/governance/canister/canister.rs

```diff
@@ -702,52 +587,34 @@ fn encode_metrics(w: &mut ic_metrics_encoder::MetricsEncoder<Vec<u8>>) -> std::i
 /// the .wasm; using `candid::export_service` here would involve unnecessary
 /// runtime computation.
 #[cfg(not(feature = "test"))]
-#[export_name = "canister_query __get_candid_interface_tmp_hack"]
-fn expose_candid() {
-    over(candid, |_: ()| include_str!("governance.did").to_string())
-}
-#[cfg(feature = "test")]
-#[export_name = "canister_query __get_candid_interface_tmp_hack"]
-fn expose_candid() {
-    over(candid, |_: ()| {
-        include_str!("governance_test.did").to_string()
-    })
+#[query(hidden = true)]
+fn __get_candid_interface_tmp_hack() -> String {
```

Comment:
Do we still need this?

Reply:
Not sure.  Would probably remove this in an unrelated PR.